src/Http/Organizer/ImportOrganizerRequestHandler.php

```diff
@@ -169,7 +169,11 @@ public function handle(ServerRequestInterface $request): ResponseInterface
             // It's not possible to catch the UniqueConstraintException that UpdateWebsite can cause here, since the
             // commands are handled async.
             /** @var string|null $commandId */
-            $commandId = $this->commandBus->dispatch($command);
+            try {
+                $commandId = $this->commandBus->dispatch($command);
+            } catch (UniqueConstraintException $e) {
+                throw ApiProblem::duplicateUrl($url->toString(), $e->getDuplicateValue());
```

Comment:
I just realized why we didn't catch this exception here before. See the comment above (lines 169-170).
I think you can remove that comment now. But it means we should really disable the async import handling in the future once we get the performance issues resolved.

Intracto must've encountered this error in the short window that we had async import handling disabled on acc.

Reply:
Good point! I did improve the comment and did mention the `synchronous_imports` config setting.